scripts/run_test_mnist

```diff
@@ -62,6 +62,12 @@ kpids="${kpids:+$kpids }$apids"
 echo "INFO: launching fedbiomed researcher"
 source $basedir/scripts/fedbiomed_environment researcher
 EXEC_TEST=$basedir/notebooks/101_getting-started.py
+if [ ! -f "$EXEC_TEST" ]
+then
+    EXEC_TEST_NB=${EXEC_TEST%.py}.ipynb
+    echo "converting ${EXEC_TEST_NB} to ${EXEC_TEST}"
+    jupyter nbconvert --to script "${EXEC_TEST_NB}"
```

Comment:
@ErwanDemairy conda environments need additional package for running `jupyter nbconvert`. Currently we get:

```
jupyter nbconvert --to script ./notebooks/101_getting-started.ipynb
...
ModuleNotFoundError: No module named 'notebook.base'
```

Reply:
Hi @ErwanDemairy 
This PR is still ongoing.
We also have #954 user issue that might be related to removing `notebooks/*.py`
I also realized removing `notebooks/*.py` breaks the documentation examples using script (`python ./notebooks/101_getting-started.py`) so we need to update them

Reply:
Should the notebooks in the documentation directories also be runnable by python, once nb-converted?
Currently, there are 3 notebooks including the autoreload snippet, e.g. they require ipython.
```bash
$ grep -r autorel *                  
docs/tutorials/security/non-private-local-central-dp-monai-2d-image-registration.ipynb:    "%load_ext autoreload\n",
docs/tutorials/security/non-private-local-central-dp-monai-2d-image-registration.ipynb:    "%autoreload 2"
docs/tutorials/monai/02_monai-2d-image-registration.ipynb:    "%load_ext autoreload\n",
docs/tutorials/monai/02_monai-2d-image-registration.ipynb:    "%autoreload 2"
docs/tutorials/scikit-learn/02_sklearn_sgd_regressor_tutorial.ipynb:    "%load_ext autoreload\n",
docs/tutorials/scikit-learn/02_sklearn_sgd_regressor_tutorial.ipynb:    "%autoreload 2"
```

Reply:
You can remove it also in tutorials - we don't need that snippet in tutorials !

Reply:
IMO, 
1. this conversation is solved.
2. the PR can be merged and closed.